__tests__/app.test.js

```diff
@@ -71,42 +71,40 @@ describe("API Articles", () => {
         .get("/api/articles/1")
         .expect(200)
         .then((response) => {
-          expect(response.body.article[0].article_id).toBe(1);
-          expect(response.body.article[0].title).toBe(
+          const article = response.body.article
+          expect(article.article_id).toBe(1);
+          expect(article.title).toBe(
             "Living in the shadow of a great man"
           );
-          expect(response.body.article[0].topic).toBe("mitch");
-          expect(response.body.article[0].author).toBe("butter_bridge");
-          expect(response.body.article[0].body).toBe(
+          expect(article.topic).toBe("mitch");
+          expect(article.author).toBe("butter_bridge");
+          expect(article.body).toBe(
             "I find this existence challenging"
           );
-          expect(response.body.article[0].created_at).toBe(
+          expect(article.created_at).toBe(
             "2020-07-09T20:11:00.000Z"
           );
-          expect(response.body.article[0].votes).toBe(100);
-          expect(response.body.article[0].article_img_url).toBe(
+          expect(article.votes).toBe(100);
+          expect(article.article_img_url).toBe(
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
           );
+            expect(article).toHaveProperty("article_id");
+            expect(article).toHaveProperty("title");
+            expect(article).toHaveProperty("topic");
+            expect(article).toHaveProperty("author");
+            expect(article).toHaveProperty("body");
+            expect(article).toHaveProperty("created_at");
+            expect(article).toHaveProperty("votes");
+            expect(article).toHaveProperty("article_img_url");
```

Comment:
The above expect statements already prove that the property exists, I would say we don't need both (keep the more specific ones). Same with the below expect statements, you are indirectly proving the data type already